gatsby-ssr.js

```diff
@@ -36,6 +36,10 @@ export const onRenderBody = ({ setHeadComponents, setPostBodyComponents }) => {
       rel="stylesheet"
       href="https://cdn.jsdelivr.net/npm/@uoguelph/uofg-styles/dist/index.css"
     />,
+
+    // YouTube and Vimeo lite options
+    <script type="module" src="https://cdn.jsdelivr.net/npm/@justinribeiro/lite-youtube@1/lite-youtube.min.js"></script>,
+    <script type="module" src="https://cdn.jsdelivr.net/npm/@slightlyoff/lite-vimeo@0.1.1/lite-vimeo.js"></script>,
```

Comment:
Consider adding a key so they're consistent with the other script links (unique keys help React do a better job of optimization and tracking elements)

Reply:
Excellent idea - adding it now (and testing out if Visual Studio Code properly replies to comments)